diesel_cli/src/main.rs

```diff
@@ -186,6 +188,59 @@ fn convert_absolute_path_to_relative(target_path: &Path, mut current_path: &Path
     result.join(target_path.strip_prefix(current_path).unwrap())
 }
 
+fn run_infer_schema(matches: &ArgMatches) {
+    let database_url = database::database_url(matches);
+    let schema_name = None;
+    let schema = diesel_infer_schema::infer_schema_for_schema_name(&database_url, schema_name);
+    let mut out = String::new();
+    let mut indent = String::new();
+    let mut skip_space = false;
+    let mut last_char = ' ';
+    
+
+    for c in schema.as_str().chars() {
```

Comment:
Just like you, I don't want to be in the business of maintaining a pretty-printer either :smiling_imp:. If you care about formatting, run it through [`rustfmt::run`](https://docs.rs/rustfmt/0.6.3/rustfmt/fn.run.html) which writes formatted code to stdout. Or the approach I took in cargo-expand which is check whether rustfmt is available on the path and pipe to it if found.